public/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>seaBay</title>
+</head>
+<body>
+  <div id="shoppingCart"></div>
+  <div id="searchBar"></div>
+  <div id="itemDetail"></div>
+  <div id="similarItems"></div>
+  <div id="reviews"></div>
+  <script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
+  <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
+  <script src="http://ec2-3-17-39-91.us-east-2.compute.amazonaws.com/"></script>
```

Comment:
You'll need to pull in everyone's bundles, not just the link. For example, to pull mine in you put
```suggestion
  <script src="http://seabay.rosscalimlim.me:3005/bundle.js"></script>
```